### DISCUSSION TITLE
Does cmdk support virtulization ?

### DISCUSSION THREAD

**Discussion by hedysnike** [pacocoursey/cmdk#211] [General]
Does cmdk support virtulization ?

**Comment by af** [comment]
From the FAQ at https://github.com/pacocoursey/cmdk:

> Virtualization? No. Good performance up to 2,000-3,000 items, though. Read below to bring your own.

**Comment by pepew-le-boss** [comment]
I have a list of all country codes (around 250 items) and the list takes around **500ms** to open.

This is not good performance in my opinion.
